Code/src/main/java/nl/utwente/group10/ui/components/ConnectionStateDependent.java

```diff
@@ -0,0 +1,22 @@
+package nl.utwente.group10.ui.components;
+
+import javafx.beans.property.IntegerProperty;
+
+/**
+ * Interface for Nodes who are dependent on their ConnectionState.
```

Comment:
Why is ConnectionState written in CamelCase? Is it a class? It seems to be an integer.